contributors/guide/owners.md

```diff
@@ -102,7 +102,8 @@ aspects of this process may be configured on a per-repo basis.
   - The PR **author** `/assign`'s all suggested **approvers** to the PR, and optionally notifies
     them (eg: "pinging @foo for approval")
   - Only people listed in the relevant OWNERS files, either directly or through an alias, can act
```

Comment:
Isn't this already covered here?  "...either directly or through an alias..."

I can see changing that minimally to:
"...either directly or through an alias as described above..."